### PR TITLE
[JP] Bump up basics/enum

### PR DESCRIPTION
**Pull request by riseshia**
Could you check this @doomspork ?

### CODE REVIEW

File: jp/lessons/basics/enum.md [hunk 1]
```diff
@@ -101,38 +111,56 @@ iex> Enum.map([0, 1, 2, 3], fn(x) -> x - 1 end)
...
 6
+
 iex> Enum.reduce(["a","b","c"], "1", fn(x,acc)-> x <> acc end)
 "cba1"
 ```
 
 ### sort
 
-コレクションをソートするのは1つ、ではなく、2つある`sort`関数を使えば簡単です。1つ目の選択として用意されているのはElixirのterm orderingを使ってソート順序を決めるというものです:
+コレクションをソートするのは1つ、ではなく、2つあるソート関数を使えば簡単です。
```
Comment (marocchino) [inline jp/lessons/basics/enum.md]: two 、?

Comment (doomspork) [reply jp/lessons/basics/enum.md]: @riseshia do you wish to address this or should I merge?

Comment (riseshia) [reply jp/lessons/basics/enum.md]: fixed 4b98122

Comment (riseshia) [reply jp/lessons/basics/enum.md]: I'm sorry to be late ;)

**Comment by doomspork** [comment]
Thank you @marocchino and @riseshia 😁

**Pull request merged**